### PR TITLE
todo in pingpong fixed

### PR DESCRIPTION
**Pull request by marija-ko**
Fixed ball going only left. Added team members

### CODE REVIEW

File: source/dagger/gameplay/ping_pong/player_scores.cpp [hunk 1]
```diff
@@ -48,8 +49,7 @@ void PlayerScoresSystem::Run()
 
     if (ballOnField == 0)
     {
-        auto& reg = Engine::Registry();
-        auto sys = Engine::GetDefaultResource<PingPongBallSystem>();
-        sys->CreatePingPongBall(s_TileSize, ColorRGBA(1, 1, 1, 1), { rand()%10 + 4,rand()%10 + 4,0 },   { 0,rand()%(s_FieldHeight / 2),0 });
+        // TODO: set speed to be random in both directions
+        CreatePingPongBall(s_TileSize, ColorRGBA(1, 1, 1, 1), { rand() % 10 - 4, rand() % 10 + 4,0 }, { 0,rand() % (s_FieldHeight / 2),0 });
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/player_scores.cpp]: try to fix this todo pls

**Review by DPrivalov — APPROVED** [review]

File: source/dagger/gameplay/ping_pong/player_scores.cpp (hunk 1 above)
Comment (marija-ko) [reply source/dagger/gameplay/ping_pong/player_scores.cpp]: It should be working now : ) it can go both left and right initially

**Review by sofijaf — APPROVED** [review]

File: source/dagger/gameplay/ping_pong/player_scores.cpp (hunk 1 above)
Comment (UbiBelETF) [reply source/dagger/gameplay/ping_pong/player_scores.cpp]: What happens if `rand()` returns `4`?

Comment (marija-ko) [reply source/dagger/gameplay/ping_pong/player_scores.cpp]: Right, it goes only vertically, fixed that

**Pull request merged**